roles/junit2json/vars/dummy_variables.yml

```
@@ -0,0 +1,44 @@
+# SPDX-License-Identifier: Apache-2.0
+---
+# Dummy variables to prevent Jinja2 template evaluation errors
+# when processing test data that contains literal {{ }} strings
+
+# This file defines placeholder values for any template-like strings
+# that appear in test data but are not meant to be evaluated as templates
+
+junit2_dummy_variables:
+  # Common test failure message placeholders
+  expectation_failed: "PLACEHOLDER_EXPECTATION_FAILED"
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)